frontend/src/pages/auth/auth.tsx

```diff
@@ -44,10 +44,14 @@ const Auth: React.FC = () => {
   };
 
   return (
-    <>
-      state: {dataStatus}
-      {getScreen(pathname)}
-    </>
+    <AuthLayout
+      formElement={
+        <>
+          state: {dataStatus}
+          {getScreen(pathname)}
+        </>
+      }
+    />
```

Comment:
Let's pass form element as children
```suggestion
    <AuthLayout>
      <>
        state: {dataStatus}
        {getScreen(pathname)}
      </>
    </AuthLayout>
```